### DISCUSSION TITLE
Solve Inverse Problems Like Image Denoising

### DISCUSSION THREAD

**Discussion by tensor2023** [neuraloperator/neuraloperator#588] [Ideas]
Thank you for your great work! I have a question: Are neural operators suitable for solving inverse problems like image denoising?

**Comment by dhpitt** [comment]
@Robertboy18 @btolooshams

**Reply by btolooshams** [reply]
Yes. You can train NO to take a noisy image and output a denoised image. Indeed, NO has a good inductive bias for denoising, given its concentration of modes in low frequencies.
